engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -57,12 +55,16 @@ object WorkflowManagerActor {
     * Data
     */
   case class WorkflowManagerData(workflows: Map[WorkflowId, ActorRef]) {
-    def idFromActor(actor: ActorRef): Option[WorkflowId] =
-      workflows.collectFirst { case (id, a) if a == actor => id }
+    def idFromActor(actor: ActorRef): Option[WorkflowId] = workflows.collectFirst { case (id, a) if a == actor => id }
     def withAddition(entry: WorkflowIdToActorRef): WorkflowManagerData =
       this.copy(workflows = workflows + (entry.workflowId -> entry.workflowActor))
-    def without(id: WorkflowId): WorkflowManagerData =
-      this.copy(workflows = workflows - id)
+
+    def withAddition(entries: List[WorkflowIdToActorRef]): WorkflowManagerData = {
```

Comment:
Turns out the singular one wasn't being used anymore anyways. However, just for you I'll give the multiple one a little dose of 
![nel](https://cloud.githubusercontent.com/assets/961771/16633474/8c8c304e-4397-11e6-9155-ff79844c33e9.png)